benchmark/scripts/Benchmark_Driver

```diff
@@ -81,42 +81,14 @@ def submit_to_lnt(data, url):
 
 def instrument_test(driver_path, test, num_samples):
     """Run a test and instrument its peak memory use"""
-    test_outputs = []
-    for _ in range(num_samples):
-        test_output_raw = subprocess.check_output(
-            ['time', '-lp', driver_path, test],
-            stderr=subprocess.STDOUT
-        )
-        peak_memory = re.match('\s*(\d+)\s*maximum resident set size',
-                               test_output_raw.split('\n')[-15]).group(1)
-        test_outputs.append(test_output_raw.split()[1].split(',') +
-                            [peak_memory])
-
-    # Average sample results
-    num_samples_index = 2
-    min_index = 3
-    max_index = 4
-    avg_start_index = 5
-
-    # TODO: Correctly take stdev
-    avg_test_output = test_outputs[0]
-    avg_test_output[avg_start_index:] = map(int,
-                                            avg_test_output[avg_start_index:])
-    for test_output in test_outputs[1:]:
-        for i in range(avg_start_index, len(test_output)):
-            avg_test_output[i] += int(test_output[i])
-    for i in range(avg_start_index, len(avg_test_output)):
-        avg_test_output[i] = int(round(avg_test_output[i] /
-                                       float(len(test_outputs))))
-    avg_test_output[num_samples_index] = num_samples
-    avg_test_output[min_index] = min(
-        test_outputs, key=lambda x: int(x[min_index]))[min_index]
-    avg_test_output[max_index] = max(
-        test_outputs, key=lambda x: int(x[max_index]))[max_index]
-    avg_test_output = map(str, avg_test_output)
-
-    return avg_test_output
-
+    test_output_raw = subprocess.check_output(
```

Comment:
We first need to get the basics working at all. The script as is, does not report average memory use at all, but a sum of memory used in all runs — results depend on `num-iterations`. The `MAX_RSS` is not being reported by the `compare_perf_tests.py` either.

I have compared two runs (`num-iterations` 3 and 20) with the fixed script on my machine and the difference in `MAX_RSS` between the runs for all tests is in 0-1% range, with the exception of following: 

Testname | Mem Diff 
----------|--------------
Array2D | 2 % 
ArrayAppendSequence | 2 %
ArrayAppend | 3%
ArrayAppendStrings | 4 %
ArrayPlusEqualSingleElementCollection | 2 %
ArraySubscript | 9 %
DictionaryBridge | 90 %
MonteCarloE | 3 %
ObjectiveCBridgeStubFromNSStringRef | 9 %
ObjectiveCBridgeStubToNSStringRef | 7 %
ObjectiveCBridgeToNSDictionary | 84 %
ObjectiveCBridgeToNSSet | 84 %
ObserverClosure | 2 %
ObserverPartiallyAppliedMethod | 3 %

It looks like there are some issues (leaks ?) with ObjC Bridging (DictionaryBridge, ObjectiveCBridgeToNSDictionary, ObjectiveCBridgeToNSSet). Let me emphasize that I was comparing run with 3 vs 20 samples, done by `Benchmark_O`. 

I don’t see what could be gained by computing average memory used over multiple runs executed by `Benchmark_Driver` — are we really expecting Swift to have varying memory consumption between runs?!? That approach resulted in the need to redo all the statistics `Benchmark_O` already does in `Benchmark_Driver`- *none of which* was implemented correctly.